prody/proteins/interactions.py

```diff
@@ -0,0 +1,3288 @@
+    resnames = ag.getResnames()
+    resnums = ag.getResnums()
+    names = ag.getNames()
+    chids = ag.getChids()
+    for k in pairList:
+        if 180-angle < float(k[-1]) < 180 and float(k[-2]) < distA:
+            aa_donor = resnames[k[0]]+str(resnums[k[0]])
+            aa_donor_atom = names[k[0]]+'_'+str(k[0])
+            aa_donor_chain = chids[k[0]]
+            aa_acceptor = resnames[k[2]]+str(resnums[k[2]])
+            aa_acceptor_atom = names[k[2]]+'_'+str(k[2])
+            aa_acceptor_chain = chids[k[2]]
+            
+            HBs_list.append([str(aa_donor), str(aa_donor_atom), str(aa_donor_chain), str(aa_acceptor), str(aa_acceptor_atom), 
+                             str(aa_acceptor_chain), np.round(float(k[-2]),2), np.round(180.0-float(k[-1]),2)])
+    
+    HBs_list = sorted(HBs_list, key=lambda x : x[-2])
+    HBs_list_final = removeDuplicates(HBs_list)
+    
+    sel_kwargs = {k: v for k, v in kwargs.items() if k.startswith('selection')}
+    HBs_list_final2 = filterInteractions(HBs_list_final, atoms, **sel_kwargs)
+    
+    LOGGER.info(("%26s   <---> %30s%12s%7s" % ('DONOR (res chid atom)','ACCEPTOR (res chid atom)','Distance','Angle')))
+    for kk in HBs_list_final2:
+        LOGGER.info("%10s%5s%14s  <---> %10s%5s%14s%8.1f%8.1f" % (kk[0], kk[2], kk[1], kk[3], kk[5], kk[4], kk[6], kk[7]))
+                                
+    LOGGER.info("Number of detected hydrogen bonds: {0}.".format(len(HBs_list_final2)))
+                
+    return HBs_list_final2   
+    
+    
+def calcChHydrogenBonds(atoms, **kwargs):
+    """Finds hydrogen bonds between different chains.
+    See more details in calcHydrogenBonds().
+    
+    :arg atoms: an Atomic object from which residues are selected
+    :type atoms: :class:`.Atomic`
+    
+    :arg distA: non-zero value, maximal distance between donor and acceptor.
+        default is 3.0.
+    :type distA: int, float
+
+    :arg angle: non-zero value, D-H-A angle (donor, hydrogen, acceptor).
+        default is 40.
+    :type angle: int, float
+    
+    :arg seq_cutoff: non-zero value, interactions will be found between atoms with index differences
+        that are higher than seq_cutoff.
+        default is 25 atoms.
+    :type seq_cutoff: int
+
+    Structure should contain hydrogens.
+    If not they can be added using addMissingAtoms(pdb_name) function available in ProDy after Openbabel installation.
+    `conda install -c conda-forge openbabel`
+    
+    Note that the angle which it is considering is 180-defined angle D-H-A (in a good agreement with VMD)
+    Results can be displayed in VMD. """
+
+    try:
+        coords = (atoms._getCoords() if hasattr(atoms, '_getCoords') else
+                    atoms.getCoords())
+    except AttributeError:
+        try:
+            checkCoords(coords)
+        except TypeError:
+            raise TypeError('coords must be an object '
+                            'with `getCoords` method')
+
+    distA = kwargs.pop('distA', 3.5)
```

Comment:
we can't pop these here if we want to provide them with **kwargs below. We'd have to provide them again separately. This could be useful if you want different default values, but probably we don't.

Shall I just remove them here?